dashboard/src/app/colors/che-color.constant.js

```diff
@@ -22,6 +22,8 @@ export class CheColorsConfig {
   '$che-purple-color': '#4E5A96',
```

Comment:
I think I've failed on the split as this file should be auto generated from the json file and then should be ignored